spec/spec_helper.rb

```diff
@@ -24,3 +26,44 @@
 
   config.order = :random
 end
+
+RSpec::Matchers.define :emit_notification do |expected_event_name|
```

Comment:
Pulled this matcher from the `delayed` gem. Makes me wonder if we should try to upstream it into `rspec-rails` or something 🤔.

Reply:
Love that idea